### ISSUE TITLE
Remove unprofessional nouns or provide a filter to

### ISSUE THREAD

**Issue by srikiran-cpn** [mrmaxguns/wonderwordsmodule#34]
**Is your feature request related to a problem? Please describe.**
When using this software in a business setting, we don't want nouns like junk, panty etc to appear. Both of these have appeared in client facing contexts.

**Describe the solution you'd like**
Two ways forward
- Remove some nouns from nounlist.txt
- Provide a filter to remove these words

**Describe alternatives you've considered**
I am considering forking and making a modification to nounslist.txt at the moment.

**Additional context**
N/A

**Comment by adunmore** [comment]
Many risky words in this project. I wouldn't recommend using it in a professional context.

**Comment by mrmaxguns** [comment]
Fair. The word lists are sourced as standard English works. In such contexts it doesn't make sense to use a whole English dictionary but perhaps a smaller list of non-objectionable words. I would advise you to provide such a dictionary: Wonderwords supports custom dictionaries and word categories.

**Comment by srikiran-cpn** [comment]
@mrmaxguns are you indicating that you would like for me to raise a PR for this? Custom dictionaries sound good

**Comment by mrmaxguns** [comment]
They exist! See https://wonderwords.readthedocs.io/en/latest/advanced.html

```python3
from wonderwords import RandomWord

fruits = ["apple", "orange", "banana", "strawberry"]
generator = RandomWord(fruit=fruits)

print(generator.word()) # ex: apple
```

An alternative would be to curate a new list directly in Wonderwords that contains non-objectionable words because I think that continuing to remove words from the default dictionaries through trial and error is counterproductive.

**Comment by srikiran-cpn** [comment]
Yes, I think removing words one by one is counterproductive. Would you like me to submit a PR for a new curated list called professional_nouns? I can imagine that it may still not be general enough for everyone's use case, so perhaps there isn't a point. You can close this issue
